tools/nl/embeddings/input/base/main_topics.csv

```diff
@@ -340,20 +340,20 @@ dc/topic/VeteransIncome,Veterans Income;Veterans Income;income of veterans;veter
 dc/topic/Visibility,Visibility;Visibility
 dc/topic/Volcano,Volcano;Volcano
 dc/topic/Voting,Voting;Voting
-dc/topic/WHOChildImmunization,Global Child Immunization;Child immunization rates;WHO child immunization

```

Comment:
Are we removing "immunization rates" too?

Reply:
yes because the "rate" causes it to show up for "global literacy rate"